doc/man3/SSL_CTX_set_num_tickets.pod

```diff
@@ -40,14 +42,30 @@ handshake then SSL_set_num_tickets() can be called again prior to calling
 SSL_verify_client_post_handshake() to update the number of tickets that will be
 sent.
 
+To issue tickets after other events (such as application-layer changes),
+SSL_new_session_ticket() is used to request that a new ticket be sent when
+it is safe to do so.  New tickets are only allowed to be sent in this manner
+after the initial handshake has completed, and only for TLS 1.3 connections.
+The ticket generation and transmission are delayed until the server is
+starting a new write operation, so that it is bundled with other application
+data being written and properly aligned to a record boundary.
+SSL_new_session_ticket() can be called more than once to request additional
+tickets be sent; all such requests are queued until and the multiple tickets
```

Comment:
"queued until". Doesn't quite make sense

Reply:
It was once "queued until it is safe to do so" but had a bad edit; fixed.